projects/admin/src/app/routes/acquisition-order-lines-route.ts

```diff
@@ -48,9 +51,9 @@ export class AcquisitionOrderLinesRoute extends BaseRoute implements RouteInterf
             key: this.name,
             label: 'Order lines',
             detailComponent: AcquisitionOrderLineDetailViewComponent,
-            canAdd: () => this._routeToolService.canAddSystemLibrarian(),
-            canUpdate: (record: any) => this._routeToolService.canUpdate(record),
-            canDelete: (record: any) => this._routeToolService.canDelete(record),
+            canAdd: () => this._routeToolService.canSystemLibrarian(),
```

Comment:
It seems too restrictive. In my sense, a `librarian` should add order lines depending of the library owner of this linked order.

Reply:
This correction is for @lauren-d